Pipfile

```diff
@@ -26,7 +26,7 @@ termcolor="*"
 junit-xml ="*"
 dpath = ">=1.5.0,<2"
 pyyaml = ">=5.4.1"
-boto3 = "*"
+boto3 = "==1.17.27"
```

Comment:
Curious: why does this improve performance?

Reply:
we are not sure. trial and error. :(

Reply:
That's an excellent question. Still trying to figure that one out

Reply:
for me today felt like the commits at: https://twitter.com/gitlost